### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by jcbhmr** [ATiltedTree/setup-rust#247]
I notice that this repo is the first hit on GitHub Actions marketplace but that there's been no commits in more than a year and there are three open Issues. 

![image](https://github.com/ATiltedTree/setup-rust/assets/61068799/7ddca19d-bbb0-445b-969b-fb471dc298f5)

I also notice that @ATiltedTree doesn't appear very active.

![image](https://github.com/ATiltedTree/setup-rust/assets/61068799/825ebab0-a486-495e-b9be-36bda3ac178e)

Bottom line: Is this project still maintained, or should I look elsewhere for a rust setup action?
